getting-started/data-setup.rst

```diff
@@ -199,8 +139,9 @@ Here are some key takeaways:
 - The Butler is the interface between data and LSST Science Pipelines processing tasks.
 - Butler repositories can be hosted on different backends, both remote and local. In this case you created a local Butler repository on your computer's filesystem.
 - Butler repositories contain raw data, calibrations, and reference catalogs. As you'll see in future tutorials, the Butler repository also contains the outputs of processing tasks.
-- Command-line tasks like :command:`ingestImages.py` and :command:`ingestCalibs.py` help you seed data into Butler repositories.
+- If you are interested in creating a butler repository with your own data, the `Community Forum`_ is the right place to search for and ask questions.
```

Comment:
Begging the question of where we are actually going to teach people how to use `butler register-instrument` and `butler ingest-raws`...